GetStream/stream-chat-android
# [2166] Add support for attachment type configuration

### 🎯 Goal

Hide files from the attachment picker that are not valid according to the upload config.

### 🧪 Testing

1. Navigate to the "Upload Configuration" section in the dashboard
2. Experiment with allowed / blocked file extensions and MIME types
3. Open our attachment picker in the app and ensure that only files matching the filters in the dashboard are displayed

**Please, reset the upload configuration to the defaults after testing**

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist
